x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -217,6 +377,9 @@ const CustomDataGridSingleRow = memo(function CustomDataGridSingleRow(
       {/* Timeline Expanded Row */}
       {canShowRowRenderer ? (
         <Cell
+          rowHeightsOptions={defaultAutoHeight}
+          /* @ts-expect-error because currently CellProps do not allow string width but it is important to be passed for height calculations   */
+          width={'100%'}
```

Comment:
So it currently "works" because the rest props are spread onto the component?

Reply:
Exactly. Here is the code in EUI: https://github.com/elastic/eui/pull/8028/files#diff-79459b26a5c68fbbf1233bd815d9f57ba81326b9632b27b3ddb3da6b0a25c92eR229